sql-plugin/src/main/scala/com/nvidia/spark/RebaseHelper.scala

```diff
@@ -48,9 +48,7 @@ object RebaseHelper {
       startTs: Long): Boolean = {
     val dtype = column.getType
     if (dtype.hasTimeResolution) {
-      // TODO - https://github.com/NVIDIA/spark-rapids/issues/1130 to properly handle
-      // TIMESTAMP_MILLIS, for use require so we fail if that happens
-      require(dtype == DType.TIMESTAMP_MICROSECONDS)
+      assert(dtype == DType.TIMESTAMP_MICROSECONDS)
```

Comment:
Should this stay as a `require`? We don't support anything other than microseconds in this method, so it would be good to make 100% sure we are still only handling micros here.

Reply:
Updated, thanks